### ISSUE TITLE
TimberTerm : different behaviour in php and twig file

### ISSUE THREAD

**Issue by twentyfortysix** [timber/timber#1159]
### Expected behavior

when I call this in index.**php**

```
$zp_term = new TimberTerm(34, 'publication_type');

var_dump($zp_term);

class Timber\Term#682 (15) {
  public $PostClass =>
  string(11) "Timber\Post"
  public $TermClass =>
  string(4) "Term"
  public $object_type =>
  string(4) "term"
  public $_children =>
  NULL
  public $name =>
  string(18) "Novinky z knihovny"
  public $taxonomy =>
  string(16) "publication_type"
  public $id =>
  int(34)
  public $ID =>
  int(34)
  public $term_id =>
  int(34)
  public $slug =>
  string(18) "novinky-z-knihovny"
  public $term_group =>
  int(0)
  public $term_taxonomy_id =>
  int(34)
  public $parent =>
  int(0)
  public $count =>
  int(5)
  public $filter =>
  string(3) "raw"
}
```

Which is what I expect.

But in **twig**:

`{% set zp_term = TimberTerm('34', 'publication_type') %}`
I receive this

```
Fatal error: Class 'publication_type' not found in /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/lib/Twig.php on line 120

Call Stack:
    0.0001     250760   1. {main}() /Library/WebServer/localhost/crq/index.php:0
    0.0002     253568   2. require('/Library/WebServer/localhost/crq/wp-blog-header.php') /Library/WebServer/localhost/crq/index.php:17
    0.2698   36336648   3. require_once('/Library/WebServer/localhost/crq/wp-includes/template-loader.php') /Library/WebServer/localhost/crq/wp-blog-header.php:19
    0.2735   36399536   4. include('/Library/WebServer/localhost/crq/wp-content/themes/Almond-milk/index.php') /Library/WebServer/localhost/crq/wp-includes/template-loader.php:75
    0.4255   37741744   5. Timber\Timber::render() /Library/WebServer/localhost/crq/wp-content/themes/Almond-milk/index.php:62
    0.4255   37741944   6. Timber\Timber::fetch() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/lib/Timber.php:332
    0.4255   37742168   7. Timber\Timber::compile() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/lib/Timber.php:317
    0.4263   37790312   8. Timber\Loader->render() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/lib/Timber.php:281
    0.4330   38740936   9. Twig_Environment->render() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/lib/Loader.php:72
    0.4832   41180720  10. Twig_Template->render() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php:347
    0.4832   41197440  11. Twig_Template->display() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:366
    0.4833   41198592  12. Twig_Template->displayWithErrorHandling() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:355
    0.4833   41198680  13. __TwigTemplate_6e8eb0a034164a10a9e1f184354ae262aba9c8cb43ebd1571814eaafa41a1ec3->doDisplay() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:387
    0.5183   41506616  14. Twig_Template->display() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php(403) : eval()'d code:27
    0.5184   41507480  15. Twig_Template->displayWithErrorHandling() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:355
    0.5184   41507696  16. __TwigTemplate_7e8d477af9e37ebab2e1bd6c020e48b0acfade94b5e0e40795957bc8cc906084->doDisplay() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:387
    0.5764   41607568  17. Twig_Template->displayBlock() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php(403) : eval()'d code:120
    0.5764   41608144  18. __TwigTemplate_6e8eb0a034164a10a9e1f184354ae262aba9c8cb43ebd1571814eaafa41a1ec3->block_content() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Template.php:167
    0.5765   41608752  19. call_user_func_array:{/Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php(403) : eval()'d code:38}() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php(403) : eval()'d code:38
    0.5765   41608824  20. Timber\Twig->Timber\{closure}() /Library/WebServer/localhost/crq/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php(403) : eval()'d code:38
```
### What version of WordPress, PHP and Timber are you using?

WordPress 4.5.3,PHP Version => 5.6.20, Timber 1.1.2

**Comment by jarednova** [comment]
Closed by #1573

**Comment by jarednova** [comment]
Thanks for reporting @twentyfortysix! (and sorry it took so long to resolve! this one got lost amongst the forest)
